src/tokens/transformation/web/createPropertyFormatter.js

```diff
@@ -19,40 +19,7 @@ const defaultFormatting = {
   suffix: ';'
 }
 
-/**
- * Creates a function that can be used to format a property. This can be useful
- * to use as the function on `dictionary.allTokens.map`. The formatting
- * is configurable either by supplying a `format` option or a `formatting` object
- * which uses: prefix, indentation, separator, suffix, and commentStyle.
- * @memberof module:formatHelpers
- * @example
- * ```javascript
- * StyleDictionary.registerFormat({
- *   name: 'myCustomFormat',
- *   formatter: function({ dictionary, options }) {
- *     const { outputReferences } = options;
- *     const formatProperty = createPropertyFormatter({
- *       outputReferences,
- *       dictionary,
- *       format: 'css'
- *     });
- *     return dictionary.allTokens.map(formatProperty).join('\n');
- *   }
- * });
- * ```
- * @param {Object} options
- * @param {Boolean} options.outputReferences - Whether or not to output references. You will want to pass this from the `options` object sent to the formatter function.
- * @param {Dictionary} options.dictionary - The dictionary object sent to the formatter function
- * @param {String} options.format - Available formats are: 'css', 'sass', 'less', and 'stylus'. If you want to customize the format and can't use one of those predefined formats, use the `formatting` option
- * @param {Object} options.formatting - Custom formatting properties that define parts of a declaration line in code. The configurable strings are: prefix, indentation, separator, suffix, and commentStyle. Those are used to generate a line like this: `${indentation}${prefix}${prop.name}${separator} ${prop.value}${suffix}`
- * @returns {Function}
- */
-function createPropertyFormatter({
-  outputReferences,
-  dictionary,
-  format,
-  formatting = {}
-}) {
+function createPropertyNameFormatter(format, formatting = {}) {
```

Comment:
`createPropertyNameFormatter` extracted from `createPropertyFormatter` so that it can be used by `formatCssAsJs` instead of making this change twice.